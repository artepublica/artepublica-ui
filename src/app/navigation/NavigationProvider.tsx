import { StatusBar } from 'expo-status-bar';
import { View } from 'react-native';

import { useTheme } from '@utils';

import NavigationContainer from './NavigationContainer.wip';
import { RootNavigator } from './RootNavigator';

function NavigationProvider(): JSX.Element {
    const { theme } = useTheme();

    return (
        <View style={{ flex: 1, overflow: 'hidden', backgroundColor: theme.background }}>
            <StatusBar translucent backgroundColor={theme.background} />
            <NavigationContainer>
                <RootNavigator />
            </NavigationContainer>
        </View>
    );
}

export default NavigationProvider;
