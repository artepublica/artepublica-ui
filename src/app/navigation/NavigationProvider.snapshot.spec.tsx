import { create } from 'react-test-renderer';

import NavigationProvider from './NavigationProvider';

describe('<NavigationProvider />', () => {
    it('default snapshot', () => {
        const json = create(<NavigationProvider />).toJSON();

        expect(json).toMatchSnapshot();
    });
});
