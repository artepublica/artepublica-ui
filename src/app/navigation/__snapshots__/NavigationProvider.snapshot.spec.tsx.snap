// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavigationProvider /> default snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": undefined,
      "flex": 1,
      "overflow": "hidden",
    }
  }
/>
`;
